website/docs/core/brands.md

```diff
@@ -19,4 +19,4 @@ This means that if you want to select a default flow based on policy, you can ju
 
 ## Branding
 
-The brand can configure the branding title (shown in website document title and several other places), and the sidebar/header logo.
+The brand can configure the branding title (shown in website document title and several other places), and the sidebar/header logo that appears in the upper left of the product interface.
```

Comment:
Suggestion: "The brand can be configured with the branding title..."  Or, actively, "You can configure the brand with the branding title..."